.github/workflows/windows-ci.yml

```diff
@@ -0,0 +1,64 @@
+name: Windows CI
+on: [pull_request,push]
+
+jobs:
+  run-windows-tests:
+    name: Build & run tests
+    runs-on: windows-2019
+
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout Code
+     
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.9.1'
+        
+    - name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v1.0.0
+      with:
+        vs-version: 16.5
+       
+    - name: Setup NuGet
+      uses: NuGet/setup-nuget@v1.0.2
+
+    - name: Check node modules cache
+      uses: actions/cache@v1
+      id: yarn-cache
+      with:
+        path: ./node_modules
+        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    - name: Install node modules
+      if: steps.yarn-cache.outputs.cache-hit != 'true'
+      run: 
+        yarn add react-native@0.62 --dev
+        yarn --pure-lockfile
+    
+    - name: yarn build
+      if: steps.yarn-cache.outputs.cache-hit == 'true'
+      run: |
+        yarn build
+        yarn tsc
+ 
+    - name: NuGet restore
+      run: nuget restore example\windows\ProgressViewExample.sln
+
+    - name: Build x64 release
+      run: msbuild .\example\windows\ProgressViewExample.sln /p:Configuration=Release /p:Platform=x64 -m
+      
+    - name: Deploy
+      shell: powershell
+      run: |
+        cd example
+        Copy-Item -Path windows\x64\Release -Recurse -Destination windows\
+        npx react-native run-windows --arch x64 --release --no-build --no-packager
```

Comment:
sorry let me rephrase: can we just replace all of this with just running yarn windows or npx react-native run-windows?

Reply:
I remember seeing some bundling errors when building release using just run-windows command, maybe that's fixed now. @TatianaKapos you can try replacing all this with `npx react-native run-windows --arch x64 --release --no-packager` and see if it works.

Reply:
hmmm yeah I'm seeing an error where it can't find the package in the release folder :/